_data/banking.yml

```diff
@@ -650,8 +650,10 @@ websites:
   - name: N26
     url: https://n26.com
     img: n26.png
-    twitter: n26
-    facebook: N26
+    tfa:
+      - sms
```

Comment:
SMS seems to only be a backup method.
```suggestion
```

Reply:
I don't think so. The documentation says (under "What if my phone doesn’t support the latest app version?") that SMS verification is possible for logins, i.e. authentication.

Reply:
![](https://d.carlgo11.com/D5DB088AF9615D/?p=CqSTKaR5ox3Y9kB)

Reply:
Right, but that is _authorization_, not _authentication_, or am I mistaken here?

Reply:
No you're right. Good catch :+1: